ALTS authentication - Cannot make examples run

Several issues using example code with tag `v1.35.x` and this document https://grpc.io/docs/languages/cpp/alts/ 

**Compilation**

```c++
using grpc_impl::experimental::AltsCredentialsOptions;
```
 Doesn't seem to exist.

```
[ 92%] Building CXX object CMakeFiles/greeter_server.dir/greeter_server.cc.o
/Users/avbenavides/git/grpc/grpc/examples/cpp/helloworld/greeter_server.cc:46:7: error: use of undeclared identifier 'grpc_impl'
using grpc_impl::experimental::AltsServerCredentialsOptions;
```

Using `using grpc::experimental::AltsServerCredentialsOptions;` compiles but then...

**Seg Fault**

Adding the server ATLS auth sample code modified as above to _helloworld_ example results in a seg fault.
```c++
...
#include <grpcpp/security/server_credentials.h>
#include <grpcpp/security/alts_util.h>
...
using grpc::experimental::AltsServerCredentials;
using grpc::experimental::AltsServerCredentialsOptions;
...


void RunServer() {
  std::string server_address("0.0.0.0:50051");
  GreeterServiceImpl service;
  
  ServerBuilder builder;
  builder.RegisterService(&service);
  auto creds = AltsServerCredentials(AltsServerCredentialsOptions());
  // Listen on the given address without any authentication mechanism.
  builder.AddListeningPort(server_address, creds);
  // Register "service" as the instance through which we'll communicate with
  // clients. In this case it corresponds to an *synchronous* service.
  
  // Finally assemble the server.
  std::unique_ptr<Server> server(builder.BuildAndStart());
  std::cout << "Server listening on " << server_address << std::endl;

  // Wait for the server to shutdown. Note that some other thread must be
  // responsible for shutting down the server for this call to ever return.
  server->Wait();
}
...
```

Running
```
$ ./greeter_server 
E0315 10:02:12.337824000 4601167360 server_secure_chttp2.cc:81]        {"created":"@1615798932.337809000","description":"No credentials specified for secure server port (creds==NULL)","file":"/Users/avbenavides/git/grpc/grpc/src/core/ext/transport/chttp2/server/secure/server_secure_chttp2.cc","file_line":55}
Server listening on 0.0.0.0:50051
Segmentation fault: 11
```

--- a/content/en/docs/languages/cpp/alts.md
+++ b/content/en/docs/languages/cpp/alts.md
@@ -13,7 +13,7 @@ code:
     #include <grpcpp/security/credentials.h>
 
     using grpc::experimental::AltsCredentials;
-    using grpc_impl::experimental::AltsCredentialsOptions;
+    using grpc::experimental::AltsCredentialsOptions;
 
     auto creds = AltsCredentials(AltsCredentialsOptions());
     std::shared_ptr<grpc::Channel> channel = CreateChannel(server_address, creds);
@@ -25,7 +25,7 @@ code:
     #include <grpcpp/server_builder.h>
 
     using grpc::experimental::AltsServerCredentials;
-    using grpc_impl::experimental::AltsServerCredentialsOptions;
+    using grpc::experimental::AltsServerCredentialsOptions;
 
     grpc::ServerBuilder builder;
     builder.RegisterService(&service);
@@ -39,7 +39,7 @@ code:
     #include <grpcpp/security/credentials.h>
 
     using grpc::experimental::AltsCredentials;
-    using grpc_impl::experimental::AltsCredentialsOptions;
+    using grpc::experimental::AltsCredentialsOptions;
 
     AltsCredentialsOptions opts;
     opts.target_service_accounts.push_back("expected_server_service_account1");

